JS/index.js

```diff
@@ -8,11 +8,52 @@ wmp.addEventListener('click', () => {
 // Header
 const signUpBtn = document.querySelector('.sign-up');
 signUpBtn.addEventListener('click', () => {
-    // console.log("Take my money!!");
     location.href = "https://water-my-plants-client.herokuapp.com/register";
 });
 
 const learnMoreBtn = document.querySelector('.learn-more');
 learnMoreBtn.addEventListener('click', () => {
     location.href = "#intro";
+});
+
```

Comment:
Great use of a constructor function and methods, also good use of array methods!